governance/default.go

```diff
@@ -150,8 +135,7 @@ type VoteStatus struct {
 }
 
 type Governance struct {
-	ChainConfig    *params.ChainConfig
-	governanceLock sync.RWMutex
```

Comment:
Don't we need this lock?

Reply:
This lock was temporarily added. And the data race problem can't be solved in this way after enabling the voting feature. In following PRs, each object which can be accessed from multiple callers will have its own lock.